pom.xml

```diff
@@ -31,10 +31,12 @@
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+            <version>2.0.2</version>
```

Comment:
versions are better to store in properties

Reply:
but in this case version is provided by spring boot parent

Reply:
fixed